__tests__/api.test.js

```diff
@@ -208,6 +196,29 @@ describe('app', () => {
           expect(msg).toBe('Invalid Path')
         })
       })
-  })
-
-})
\ No newline at end of file
+      })
+      
+  describe("5. GET /api/reviews/:review_id", () => {
+    test('should return a review object with comment_count added on', () => {
+      return request(app)
+        .get('/api/reviews/2')
+        .expect(200)
+        .then(({ body }) => {
+          const { review } = body;
+              expect(review).toEqual(
+                expect.objectContaining({
+                  category: expect.any(String),
+                  created_at: expect.any(String),
+                  designer: expect.any(String),
+                  owner: expect.any(String),
+                  review_body: expect.any(String),
+                  review_id: expect.any(Number),
+                  review_img_url: expect.any(String),
+                  title: expect.any(String),
+                  votes: expect.any(Number),
+                  comment_count: expect.any(String)
+                }))

```

Comment:
this looks good, but I would be more specific with your test, if you are searching for api/reviews/2 then you know what the review will look like 

ie, review_id should be 2 etc

have a look at the test data or what you are getting back as review here in your test to make it stricter :)